README.md

```diff
@@ -110,6 +110,8 @@ $ wrangler publish
 
   You can also [use environment variables](https://developers.cloudflare.com/workers/tooling/wrangler/configuration/) to configure these values.
 
+  For more information on Cloudflare API keys and tokens [please visit](https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys).
```

Comment:
Generally we try to link directly to Workers docs in Wrangler - perhaps adding links to API token and global API key above (https://developers.cloudflare.com/workers/quickstart#api-token and https://developers.cloudflare.com/workers/quickstart#global-api-key)